### DISCUSSION TITLE
build.gradle.kts Error and Imports

### DISCUSSION THREAD

**Discussion by awesome-ux** [Minestom/Minestom#2280] [Help] [answered]
Hey everyone,

I try to work with Minestom my first time,
But I have a lot of error

There is my build.gradle.kt (https://haste.devstorage.eu/O1hs8hjwEr)

And that is my error:

*
Test:main: Could not find com.github.Minestom:Minestom:1f34e60ea6.
Searched in the following locations:
  - https://repo.maven.apache.org/maven2/com/github/Minestom/Minestom/1f34e60ea6/Minestom-1f34e60ea6.pom
  - https://jitpack.io/com/github/Minestom/Minestom/1f34e60ea6/Minestom-1f34e60ea6.pom
Required by:
    project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
*

**Comment by cosrnic** [comment, marked as answer]
you should be using `implementation("net.minestom:minestom-snapshots:1f34e60ea6")` but you also need to tell gradle to use java 21, something like this should work
```
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(21)
    }
}
```
